app/tests/test_admin_page.py

```diff
@@ -220,11 +221,26 @@ def _create_asset_minimal(self, item):
 
         return asset, data
 
-    def _create_asset(self, item, extra=None):
+    def _create_asset(self, item, extra=None, large_file=False):
         start = time.time()
-        filecontent = b'mybinarydata'
-        filelike = BytesIO(filecontent)
-        filelike.name = 'testname.tiff'
+        if large_file:
+            logger.warning(
+                "Running test for admin page with very large asset file. "
+                "This can take a while (about 1.5 minutes), sorry."
```

Comment:
as you mentioned.... maybe just skipping in the default config

Reply:
Done :-)
But I set the default to `True`, so that one has to explicitly disable it in `.env.local`, just to be on the safe side.